### PR TITLE
Initial python basics commit

### PR DESCRIPTION
**Pull request by lucywyman**

### CODE REVIEW

**Comment by lucywyman** [comment]
@ElijahCaine @jordane  How does that look?

File: programming-basics/exercise.rst [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+Once you've successfully run the first exercise from
+the REPL as a script, here are a few more exercises to 
+try:
+
+#. Create a list (as long as you want!), then programmatically
+   find how long it is.
+#. Append a new number to the end of your list and make sure
+   your algorithm works.
+#. Define a function :code:`length()` that takes one argument
```
Comment (ElijahCaine) [inline programming-basics/exercise.rst]: I didn't even know ``:code:`` was a thing.

File: programming-basics/exercise.rst [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+#. Print your name to a file
+#. Read your name from a file
+
+If you want to keep challenging yourself, check out `Project Euler`_ coding
+puzzles.
+
+.. _Project Euler: https://projecteuler.net/archives
+
+
+Next we'll look at writing python scripts. A script
```
Comment (ElijahCaine) [inline programming-basics/exercise.rst]: It's not a *problem* but your ~60 character/line style is weird.

**Comment by ElijahCaine** [comment]
Looks good. Ignore my pedantic comments.

**Pull request merged**